WordPress/gutenberg
# [RNMobile] Include `support inserter` in Column Block

## Description

This PR brings adjustments need to allow using `support inserter` inside Column Block.

Please refer to:
[Allow support inserter in Columns block PR](https://github.com/WordPress/gutenberg/pull/20502)
[Column Block PR](https://github.com/WordPress/gutenberg/pull/19013)
[design note](https://github.com/WordPress/gutenberg/pull/19013#issuecomment-596734085)

## How has this been tested?




Checking all Column Block PR testing requirements ([see Column PR description](https://github.com/WordPress/gutenberg/pull/19013#issue-350696448))

List of changes:
- movers buttons change directions depending on number of Column in row
- add appender which automatically adds new Column to the end of InnerBlock of Columns block
- Column placeholder is visible only when Column block is selected
- enable Inserter which allows to select only Column block
- remove separator line when adding Column via Inserter
- bring back remove button on mobile toolbar
- removing last Column remove also the parent ( Columns ) block
- limit Stepper to min value of 1
- remove max limit in Stepper (here's the trick the max value for Stepper depends on Column count (it equals `columnCount + 1` so from user perspective there is no limit)

Most of above changes was replicate from [Buttons PR](https://github.com/WordPress/gutenberg/pull/20191) and was discussed and approved by @iamthomasbishop. Column block is quite similar in those cases

## Types of changes





New feature  - Change Columns/Column blocks behaviour according to design after support inserter enable.

## Checklist:
